server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -119,6 +122,124 @@ const orderMutations = {
       return models.Order.findOne(query);
     },
   },
+  updateOrder: {
+    type: Order,
+    description: "Update an Order's amount, tier, payment method, or frequency",
+    args: {
+      order: {
+        type: new GraphQLNonNull(OrderReferenceInput),
+        description: 'Object matching the OrderReferenceInput (id)',
+      },
+      amount: {
+        type: GraphQLInt,
```

Comment:
Should be an `Amount` (type already existing). Not sure if that works well as Input or if we need to create an `AmountInput`. Only create `AmountInput` if `Amount` doesn't work.